hudi-support/extensions/README.md

```diff
@@ -1,11 +1,22 @@
 # Hudi Extensions
-## When should you use them?
+## Writer Extensions
+### When should you use them?
 The Hudi extensions provide the ability to add field IDs to the parquet schema when writing with Hudi. This is a requirement for some engines, like BigQuery and Snowflake, when reading an Iceberg table. If you are not planning on using Iceberg, then you do not need to add these to your Hudi writers.
-## How do you use them?
+### How do you use them?
 1. Add the extensions jar (`hudi-extensions-0.1.0-SNAPSHOT-bundled.jar`) to your class path.  
 For example, if you're using the Hudi [quick-start guide](https://hudi.apache.org/docs/quick-start-guide#spark-shellsql) for spark you can just add `--jars hudi-extensions-0.1.0-SNAPSHOT-bundled.jar` to the end of the command. 
 2. Set the following configurations in your writer options:  
    `hoodie.avro.write.support.class: io.onetable.hudi.extensions.HoodieAvroWriteSupportWithFieldIds`  
    `hoodie.client.init.callback.classes: io.onetable.hudi.extensions.AddFieldIdsClientInitCallback`  
    `hoodie.datasource.write.row.writer.enable : false` (RowWriter support is coming soon)  
-3. Run your existing code that use Hudi writers
\ No newline at end of file
+3. Run your existing code that use Hudi writers
+
+## Hudi Streamer Extensions
+### When should you use them?
+If you want to use OneTable with Hudi [streaming ingestion](https://hudi.apache.org/docs/hoodie_streaming_ingestion) to sync each commit into other table formats.
+### How do you use them?
+1. Add the extensions jar (`hudi-extensions-0.1.0-SNAPSHOT-bundled.jar`) to your class path.
+2. Add `io.onetable.hudi.sync.OneTableSyncTool` to your list of sync classes
+3. Set the following configurations based on your preferences:
```

Comment:
same above configs should be set here also right ? can we add a line about it ?

Reply:
Above configs are only required for cases where you're writing to an Iceberg target and need to keep the field IDs in the parquet schema

Reply:
Right but they would still need to add that config as one of the steps here if they want Iceberg as a target right, can we just add that line to be more explicit ?

Reply:
It's not related to the hudi sync tool though. It's required on the data writer. I think it is fine as it is.